upgrades.js

```diff
@@ -42,28 +70,27 @@ const upgradeAntimatterCap1 = () => {
 const upgradeAnnihilationSpeed1 = () => {
 	let unlockElement = document.getElementById("upgrade_annihilation_speed_1");
 
-	if (data["Light"] >= .05) {
-		data["AnnihilationSpeed"] *= 2;
-		data["Light"] -= .05;
+	if (data["resources"]["light"] >= 0.05) {
+		data["annihilation"]["speed"] = 0.02;
+		data["resources"]["light"] -= 0.05;
+		data["upgrades"]["annihilationSpeed1"] = true;
 		unlockElement.style.display = "none";
 	}
 
 	// Unlock multiplier button
 	document.getElementById("upgrade_annihilation_multiplier").style.display = "block";
+	data["visible"]["annihilationMultiplier"] = true;
 	updateDisplay();
 }
 
 /// Multiplier stuff
 const upgradeAnnihilationMultiplier = () => {
-	console.log(data["AnnihilationMultiplierCount"]);
-
-	const cost = fib[data["AnnihilationMultiplierCount"]] * 1;
+	const cost = fib[data["annihilation"]["multiplier"]];
 
 	console.log(cost);
```

Comment:
why are we logging

Reply:
probably left in as debug, I'll go and remove any of these that don't need to be there